### PR TITLE
[ENH] Add top-level note in Nipoppy pages about new Read the Docs website 

### LINKED ISSUE #192: Point to Nipoppy docs for new Nipoppy with CLI
### Description

Add link to [nipoppy.readthedocs.io](http://nipoppy.readthedocs.io/) :)

### Relevant documentation pages

_No response_

### Additional details

_No response_

### PR DESCRIPTION
**Pull request by michellewang**
- Closes #192 

Changes proposed in this pull request:

- Added a "Note" block at the top of every Nipoppy page, for example:
  Screenshot 2024-05-07 at 7 41 54 PM

  

## Checklist
_Please leave checkboxes empty for PR reviewers_

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`) _see our [Contributing Guidelines](https://neurobagel.org/contributing/CONTRIBUTING/#pull-request-guidelines) for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [ ] Checks pass

### CODE REVIEW

**Comment by michellewang** [comment]
Markdown link check failure seems to be due to other Neurobagel pages -- pinging @surchs @alyssadai  @rmanaem

**Comment by surchs** [comment]
md links are failing because #189 is not merged yet - I think you can ignore that @michellewang.

File: docs/nipoppy/configs.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+{%
+    include-markdown "nipoppy/cli_note.md"
+%}
```
Comment (surchs) [inline docs/nipoppy/configs.md:3]: neat!

File: docs/nipoppy/cli_note.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+!!! note
```
Comment (surchs) [inline docs/nipoppy/cli_note.md:1]:
```suggestion
!!! note "Nipoppy docs are moving"
```

:cherries: or something similar - could use the title to make the point.

**Review by surchs — APPROVED** [review]
Looks good @michellewang! 

:cook:

**Pull request merged**